runtime/guard-jwt/src/main/java/io/aklivity/zilla/runtime/guard/jwt/internal/JwtGuardHandler.java

```diff
@@ -54,39 +63,43 @@ public class JwtGuardHandler implements GuardHandler
     private final Long2ObjectHashMap<JwtSessionStore> sessionStoresByContextId;
 
     public JwtGuardHandler(
-        JwtOptionsConfig options,
-        LongSupplier supplyAuthorizedId)
+            GuardConfig guard,
+            LongSupplier supplyAuthorizedId)
```

Comment:
Recommend reverting this change in parameter types, and adding `readURL` as an additional parameter instead.

Reply:
I actually use the `GuardConfig` parameter to access both the `readURL` and `jsonb` transient fields. Should I also add `jsonb` as a parameter here?

Reply:
Yes, that works.